### DISCUSSION TITLE
JioAirfiber support

### DISCUSSION THREAD

**Discussion by BlueNecko** [JFC-Group/JF-Customisation#42] [Q&A] [answered]
I stumbled upon this wonderfull repo yesterday and i read the research that that has been done on the jiofiber. I myself am very keen on revealing more secrets that my router hides at the moment. I have reached a wall for now as getting the encryption key is near impossible for me without any further help. I tried to find as much similarities between the normal and the new Airfiber but in terms of software and commands a lot has changed. A totally different mess now. I would love to hear your thoughts about extending this reseach towards JioAirfiber too. As that is more locked to it's core if anything else.

**Comment by itsyourap** [comment]
@pokewizardSAM 

I have had a very little time with AirFiber but what I remember of it is that when you download the config backup file, the config backup file is an archive which contains the actual encrypted config file and an encrypted key.

Now, to decrypt the actual config file, we need the decrypted key. 
Now we have the encrypted key but in order to decrypt it, we need to understand what encryption type has been performed on the key which is not possible since we don't have the firmware.

If someone can get (sniff) the firmware URL before firmware updates, we can easily investigate stuffs.

_P.S. - Note that everything I said may not be accurate since I used AirFiber for very brief period of time_

**Reply by BlueNecko** [reply]
So far it seems that sniping the firmware updates is our only option. And more over what method was used to sniff the normal jiofiber's update urls. Something which intercepts the traffic between the modem and jio servers ?? 

This time the router is quite a bit more difficult to open as there is a paper cover on top that needs to be removed to disassemble, and might cause issues in future claims. 

I tried many of the possible combinations and requests based on past data on this repo. Lots of which don't work for this one. Many pages are hidden too in static.js

**Reply by itsyourap** [reply]
Originally, the OG sniffed the STB's update url using a Proxy Server method. 
From the STB firmware URL, we guessed the ONT update URL and it just worked.

**Reply by BlueNecko** [reply]
@itsyourap How different if jio airfiber's config encryption method. After a long time it peeked my interests again as community has not shown any new progress for this shitty locked dabba. My observations are as follows. 

1. The backup file we make of the config is universally usable even if the router is reset and hasn't connected to the internet to synchronize, which implies that there is some master key inbuilt in the router ??
2. The naming of the config backups has also been changed, previously it was ```*.enc``` but now it seems to have been changed to ```*_enc.tar.gz``` , so is the fill differently packaged now or it just to throw us off ??
3. wan packet capture while doing a reset also leaves no traces of connecting to anything noteworthy.

**Comment by itsyourap** [comment, marked as answer]
Look at https://github.com/JFC-Group/AF-Customisation

**Reply by BlueNecko** [reply]
well @sammyXFR @BlueNecko are my aliases ^_____^
